src/sentry/static/sentry/app/components/issues/snoozeAction.tsx

```diff
@@ -11,19 +11,23 @@ const Snooze = {
   '24HOURS': 60 * 24,
 };
 
-class SnoozeAction extends React.Component {
+type SnoozeActionProps = {
+  disabled: boolean;
+  onSnooze: Function;
```

Comment:
Could we be more specific here on the function signature?

Reply:
That's a good call, I should also probably convert those times to an enum while I'm here as well.

Reply:
Enums are a good idea :+1: